dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java

```diff
@@ -0,0 +1,182 @@
+package com.dotmarketing.startup.runonce;
+
+import static org.junit.Assert.assertFalse;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05225RemoveLoadRecordsToIndexTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void testUpgradeTaskShouldPass() throws DotDataException, SQLException {
+
+        createLoadRecordsToIndexProcedure();
+        final Task05225RemoveLoadRecordsToIndex task = new Task05225RemoveLoadRecordsToIndex();
+        task.executeUpgrade();
+        assertFalse(procedureExists());
+
+    }
+
+    private void createLoadRecordsToIndexProcedure() throws SQLException {
+        String script = null;
+
+        switch (DbConnectionFactory.getDBType()){
+            case "MySQL":
+                script = getMySQLScript();
+                break;
+
+            case "PostgreSQL":
+                script = getPostgreSQLScript();
+                break;
+
+            case "Oracle":
+                script = getOracleScript();
+                break;
+
+            case "Microsoft SQL Server":
+                script = getSQLServerScript();
+                break;
+        }
+
+        final DotConnect dotConnect = new DotConnect();
+        dotConnect.executeStatement(script);
+
+    }
+
+    private String getOracleScript() {
+        return "CREATE OR REPLACE FUNCTION load_records_to_index(server_id VARCHAR2, records_to_fetch NUMBER, priority_level NUMBER)\n"
+                + "   RETURN types.ref_cursor IS\n"
+                + " cursor_ret types.ref_cursor;\n"
+                + " data_ret reindex_record_list;\n"
+                + "BEGIN\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "BEGIN\n" appears 4 times in this file; the first occurrence is on line 62](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)